### PR TITLE
Updated dependancies. Improved package.json info

### PR DESCRIPTION
**Pull request by gorilla-devs**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,14 +20,39 @@
   "author": "shamofu",
   "license": "MIT",
   "build": {
+    "productName": "shamofu",
+    "appId": "org.shamofu.electronReactParcelBoilerplate",
```
Comment (shamofu) [inline package.json]:
```suggestion
    "appId": "org.shamofu.ElectronReactParcel",
```
This is just my personal preference.

File: package.json [hunk 2]
```diff
@@ -20,14 +20,39 @@
...
+    "linux": {
+      "target": [
+        "zip"
+      ],
+      "category": "Development",
+      "artifactName": "${productName}-linux.${ext}"
+    },
+    "directories": {
+      "buildResources": "resources",
+      "output": "release"
```
Comment (shamofu) [inline package.json]: Then [this line](https://github.com/shamofu/electron-react-parcel-boilerplate/blob/d9ff5abc63ca86449cea4d22f0b48fda52dc47c4/.gitignore#L2) can be removed.

File: package.json [hunk 3]
```diff
@@ -20,14 +20,39 @@
...
+    },
+    "linux": {
+      "target": [
+        "zip"
+      ],
+      "category": "Development",
+      "artifactName": "${productName}-linux.${ext}"
+    },
+    "directories": {
+      "buildResources": "resources",
```
[the remark this answers is not in the document]
Comment (gorilla-devs) [reply package.json]: Yeah sorry, this was my mistake

**Comment by gorilla-devs** [comment]
Fixed your requested changes. I added some more fixes, I hope they're welcome. otherwise I'll pull them back

**Comment by shamofu** [comment]
Many thanks!

**Pull request merged**